pkg/bot/teams.go

```diff
@@ -218,8 +217,13 @@ func (b *Teams) processActivity(w http.ResponseWriter, req *http.Request) {
 
 			msgPrefix := fmt.Sprintf("<at>%s</at>", b.BotName)
 			msgWithoutPrefix := strings.TrimPrefix(consentCtx.Command, msgPrefix)
-			msg := strings.TrimSpace(msgWithoutPrefix)
-			e := b.executorFactory.NewDefault(b.IntegrationName(), newTeamsNotifMgrForActivity(b, activity), true, msg)
+
+			ref, err := b.getConversationReferenceFrom(activity)
+			if err != nil {
+				return schema.Activity{}, fmt.Errorf("while getting conversation reference: %w", err)
+			}
+
+			e := b.executorFactory.NewDefault(b.IntegrationName(), newTeamsNotifMgrForActivity(b, ref), true, activity.ChannelID, b.bindings.Executors, msgWithoutPrefix)
 			out := e.Execute()
 
 			actJSON, _ := json.MarshalIndent(turn.Activity, "", "  ")
```

Comment:
**nit** why the error is ignored 🤔 ?

Reply:
That was not me 😄 But I will handle the error 👍